src/main/java/io/candydoc/plugin/save_ubiquitous_language/file/UbiquitousLanguageAsFile.java

```diff
@@ -27,13 +27,17 @@ public abstract class UbiquitousLanguageAsFile implements UbiquitousLanguageRepo
   public void saveAll(Set<UbiquitousLanguage> ubiquitousLanguages) {
     try {
       Files.createDirectories(destinationFolder);
+      Manifest manifest = new Manifest(serializer);
 
       for (UbiquitousLanguage ubiquitousLanguage : ubiquitousLanguages) {
         UbiquitousLanguageDto dto = UbiquitousLanguageDto.from(ubiquitousLanguage);
         String fileName = dto.getName().toLowerCase().replace(" ", "_");
         serializer.writeValue(destinationFolder.resolve(fileName + "." + extension).toFile(), dto);
+
+        manifest.addFile(destinationFolder.resolve(fileName + "." + extension));
       }
```

Comment:
maybe extract some methods.
```suggestion
      for (UbiquitousLanguage ubiquitousLanguage : ubiquitousLanguages) {
                manifest.addFile(writeToFile(ubiquitousLanguage));
      }
            
    // some private methods 
    
    private Path writeToFile(UbiquitousLanguage ubiquitousLanguage) throws IOException {
        UbiquitousLanguageDto dto = UbiquitousLanguageDto.from(ubiquitousLanguage);
        Path resolve = resolveTargetFile(dto.getName());
        serializer.writeValue(resolve.toFile(), dto);
        return resolve;
    }
    
    private Path resolveTargetFile(String name) {
        String fileName = name.toLowerCase().replace(" ", "_");
        Path resolve = destinationFolder.resolve(fileName + "." + extension);
        return resolve;
    }
```

Reply:
if the stream is supported, you can replace loop with a stream and why not the use of the parallel stream feature for the IO